app/konnect/network.md

```diff
@@ -41,17 +41,32 @@ Data plane nodes initiate the connection to the {{site.konnect_short_name}} cont
 
 To let a data plane node request and receive configuration, and send telemetry data,
-add the following hostnames to the firewall allowlist:
-
-* `cloud.konghq.com`: The {{site.konnect_short_name}} platform.
-* `<region>.api.konghq.com`: The {{site.konnect_short_name}} API.
-  Necessary if you are using decK in your workflow, decK uses this API to access and apply configurations.
-  Region can be `us`, `eu`, or `global`.
-* `<controlPlaneId>.<region>.cp0.konghq.com`: Handles configuration for a control plane.
-  Data plane nodes connect to this host to receive configuration updates.
-  This hostname is unique to each organization and control plane.
-* `<controlPlaneId>.<region>.tp0.konghq.com`: Gathers telemetry data for a control plane.
-  This hostname is unique to each organization and control plane.
+add the following hostnames to the firewall allowlist (depending on the [geographic regions](/konnect/regions) you use).
+
+### US geo
+
+| Hostname      | Description |
+|:----------|:----------|
+| `cloud.konghq.com`    | The {{site.konnect_short_name}} platform. |
+| `global.api.konghq.com` | The {{site.konnect_short_name}} API for platform authentication, identity, permissions, teams, and organizational entitlements and settings. |
+| `us.api.konghq.com` | The {{site.konnect_short_name}} API for the US region. Necessary if you are using decK in your workflow, decK uses this API to access and apply configurations. |
+| `PORTAL_ID.us.portal.konghq.com` | The URL for the Dev Portal in the US region. |
+| `CONTROL_PLANE_ID.us.cp0.konghq.com` | Handles configuration for a control plane in the US region. Data plane nodes connect to this host to receive configuration updates. This hostname is unique to each organization and control plane. |
+| `CONTROL_PLANE_ID.us.tp0.konghq.com` | Gathers telemetry data for a control plane in the US region. This hostname is unique to each organization and control plane. |
+
+
+### EU geo
+
+| Hostname      | Description |
+|:----------|:----------|
+| `cloud.konghq.com`    | The {{site.konnect_short_name}} platform. |
+| `global.api.konghq.com` | The {{site.konnect_short_name}} API for platform authentication, identity, permissions, teams, and organizational entitlements and settings. |
+| `eu.api.konghq.com` | The {{site.konnect_short_name}} API for the EU region. Necessary if you are using decK in your workflow, decK uses this API to access and apply configurations. |
+| `PORTAL_ID.eu.portal.konghq.com` | The URL for the Dev Portal in the EU region. |
+| `CONTROL_PLANE_ID.eu.cp0.konghq.com` | Handles configuration for a control plane in the EU geo. Data plane nodes connect to this host to receive configuration updates. This hostname is unique to each organization and control plane. |
+| `CONTROL_PLANE_ID.eu.tp0.konghq.com` | Gathers telemetry data for a control plane in the EU geo. This hostname is unique to each organization and control plane. |
+
```

Comment:
@smritikjaggi I didn't add a section for AU geo hostnames because I wasn't sure if they were similar or different from US/EU. Do you know what the hostnames they should allowlist are?

Reply:
@cloudjumpercat , @smritikjaggi  - we also have some network information included on [/api/index.md](https://deploy-preview-6499--kongdocs.netlify.app/konnect/api/#base-urls) , wonder if we should maintain source in network.md and just include/link to /api/index.md?

Reply:
The geo hostnames for AU follow the same pattern as that of US/EU. I like Amy's idea of maintaining the info in one place and then referencing it everywhere else.